exports.go

```diff
@@ -7,6 +7,7 @@ import "strconv"
 type ExportList []*papi.IsiExport
 type Export *papi.IsiExport
 
+// Gets a list of all exports on the cluster
 func (c *Client) GetIsiExports() (ExportList, error) {
 	exports, err := c.api.GetIsiExports()
 	if err != nil {
```

Comment:
Something weird with the browser I think. When I view this in IE the bracket on line 14 does not line up with the return statement on line 16, but when I view it in Firefox, it does.